CGVectorAdditions.h

```diff
@@ -164,6 +167,12 @@ CGVectorAngle(CGVector vector)
 }
 
 CG_INLINE CGFloat
+CGVectorAngleSK(CGVector vector)
+{
+    return fmod(M_PI * 2.0 - atan2(vector.dy, vector.dx) + M_PI_2, M_PI * 2.0);
```

Comment:
You can use `M_2_PI` instead of `M_PI * 2.0` here. Make this change and I'll be happy to merge.